docs/en/ingest-management/release-notes/release-notes-8.8.asciidoc

```diff
@@ -0,0 +1,226 @@
+* Added a specification file for the link:https://www.elastic.co/observability/universal-profiling[Universal Profiling Symbolizer] {agent-pull}2401[#2401]
+* Added a specification file for the link:https://www.elastic.co/observability/universal-profiling[Universal Profiling Collector] {agent-pull}2407[#2407]
+* Added support to specify the {fleet-server} `service_token` through a file specified with `service_token_file` {agent-pull}2424[#2424]
+* Added support to specify custom base paths during {agent} installation {agent-pull}2500[#2500]
+
+[discrete]
+[[enhancements-8.8.0]]
+=== Enhancements
+
+{fleet}::
+* Added overview dashboards in fleet {kibana-pull}154914[#154914]
+* Added raw status to Agent details UI {kibana-pull}154826[#154826]
+* Added support for dynamic_namespace and dynamic_dataset {kibana-pull}154732[#154732]
+* Added the ability to show pipelines and mappings editor for input packages {kibana-pull}154077[#154077]
+* Added placeholder to integration select field {kibana-pull}153927[#153927]
+* Added the ability to show integration subcategories {kibana-pull}153591[#153591]
+* Added the ability to create and update the package policy API return 409 conflict when names are not unique {kibana-pull}153533[#153533]
+* Added the ability to display policy changes in Agent activity {kibana-pull}153237[#153237]
+* Added the ability to display errors in Agent activity with link to Logs {kibana-pull}152583[#152583]
+* Added support for select type in integrations {kibana-pull}152550[#152550]
+* Added the ability to make spaces plugin optional {kibana-pull}152115[#152115]
+* Added proxy ssl key and certificate to agent policy {kibana-pull}152005[#152005]
+* Added `_meta` field `has_experimental_data_stream_indexing_features` {kibana-pull}151853[#151853]
+* Added the ability to create templates and pipelines when updating package of a single package policy from type integration to input {kibana-pull}150199[#150199]
+* Added user's secondary authorization to Transforms {kibana-pull}154665[#154665]
+* Added support for the Cloud Defend application to {agent} {fleet-server-pull}2477[#2477]
+* Disabled signature validation in {agent} so that only {endpoint-sec} validates policies and actions {fleet-server-pull}2562[#2562]
+
+{fleet-server}::
+* Replaced upgrade expiration and `minimum_execution_duration` with rollout_duration_seconds` {fleet-server-pull}2243[#2243]
+* Added a `poll_timeout` attribute to check in requests that the client can use to inform {fleet-server} of how long the client will hold the polling connection open for {fleet-server-pull}2491[#2491] {fleet-server-issue}2337[#2337]
+* Added a `memory_limit` configuration setting to help prevent OOM errors {fleet-server-pull}2514[#2514]
+
+{agent}::
+* Make download of {agent} upgrade artifacts asynchronous during Fleet-managed upgrade {agent-pull}2205[#2205] {agent-issue}1706[#1706]
+* Make the language used in CLI commands more consistent {fleet-server-pull}2496[#2496]
```

Comment:
We can add an entry for https://github.com/elastic/elastic-agent/pull/2518

```
* Diagnostics now preserve the Elastic Agent upgrade watcher logs, allowing upgrades that were rolled back to be debugged {agent-pull}2518[#2518] {agent-issue}2262[#2262]

```

Reply:
Can also add one for https://github.com/elastic/elastic-agent/pull/2490

```
* Fix install on MacOS by ensuring that that /usr/local/bin exists {agent-pull}2490[#2490] {agent-issue}2487[#2487]
```

Reply:
Edit: second suggestion was in 8.7.1

Reply:
I've add PR2518 to the 8.7.1 Agent "enhancements" via [35569f1](https://github.com/elastic/ingest-docs/pull/219/commits/35569f1558b8b5ac30d1b1d0897feb5b1165c329)